tests/base.py

```diff
@@ -334,7 +334,7 @@ def setUp(self):
         settings.USE_EMAIL = False
 
 
-class AdminTestCase(DbTestCase, AdminAppTestCase, UploadTestCase, MockRequestTestCase):
```

Comment:
Does this mean we can remove [`AdminAppTestCase`](https://github.com/CenterForOpenScience/osf.io/blob/349e0e061372413688133b48562799addb6524b6/tests/base.py#L252-L255) entirely?